doc/cygnus-ngsi/installation_and_administration_guide/backends_as_sth.md

```diff
@@ -0,0 +1,40 @@
+#<a name="top"></a>Backends as short-term historics
+Backends are used by Cygnus NGSI as "infinite" context data repositories. More and more data is appended to files, tables and collections as data flows from a NGSI source. Such a data flow may never end, thus, insertions may never end too, exhausting the available storing resources.
+
+Therefore, it is important to provide mechanisms in charge of controlling how much data is stored in the persistence backends, removing old data in favour of new one, resulting in some kind of short-term historic implementation. 
+
+##<a name="top"></a>How it works
+There are two approaches when deciding which data must be removed from existent historics:
+
+* By <b>capping</b> data "records"(*) once certain size limit has been reached. In other words, to control only the last N records are stored.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1392/commits/dcbac8a77cdfa1c2d71513679c6bf0ae71e812c8